pkg/cache/task.go

```diff
@@ -42,14 +43,15 @@ type Task struct {
 	pod            *v1.Pod
 	kubeClient     client.KubeClient
 	schedulerApi   api.SchedulerApi
+	schedulerCache *external.SchedulerCache
 	nodeName       string
 	sm             *fsm.FSM
 	lock           *sync.RWMutex
 }
 
-func newTask(tid string, app *Application, client client.KubeClient, schedulerApi api.SchedulerApi, pod *v1.Pod) Task {
+func newTask(tid string, app *Application, client client.KubeClient, schedulerApi api.SchedulerApi, cache *external.SchedulerCache, pod *v1.Pod) Task {
```

Comment:
cache is nil when calling this from task.go#createTaskFromPod.
```
task := newTask(string(pod.UID), app, client, scheduler, nil, pod)
```

Reply:
this is fixed in latest patch